code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -126,7 +126,7 @@ GLOBAL_LIST_EMPTY(antagonists)
 
 //button for antags to review their descriptions/info
 /datum/action/antag_info
-	name = "Open Antag Information:"
+	name = "Open Special Role Information:"
```

Comment:
nitpicking but what if instead it was named ``"Open [role] Information"``?

Reply:
sure